docs/list-available-operators.md

```diff
@@ -0,0 +1,23 @@
+# How do I list Operators available to install?
+
+There is an extension API in OLM named `PackageManifest` that contains information existing `CatalogSources`, which is essentially a repository of CSVs, CRDs, and packages that define an application in the cluster. By querying that API, you can see the list of available operators.
```

Comment:
...named `PackageManifest` that contains information **ABOUT**  existing `CatalogSources`...

Reply:
you use the word `application` instead of `operator`. Is that intended?